src/System.Text.Json/tests/Serialization/PropertyVisibilityTests.cs

```diff
@@ -81,19 +123,15 @@ public string GetMyString()
             }
         }
 
-        public class ClassWithPrivateSetterAndGetter
+        public class ClassWithIgnoreAttributeProperty
```

Comment:
We should add more test cases to capture various edges.

Reply:
Yes. Thanks. I added a couple variations.